packages/client/src/giscus.tsx

```diff
@@ -15,7 +15,7 @@ export interface GiscusProps {
 
 let GiscusKeyInc = 0;
 
-const Giscus: FC<GiscusProps> = ({ category, categoryId, repo, repoId }) => {
+const Giscus = ({ category, categoryId, repo, repoId }: GiscusProps): ReactElement | null => {
```

Comment:
The `ReactElement | null` return type is not necessary since TS will check internally for it as instantiation of the component

But that works as it is also 👍🏼

Reply:
It's always to explicitly provide return type as if component returns `undefined` it will provoke runtime error